### ISSUE TITLE
gatsby-image 插件不支持网络图片吗？

### ISSUE THREAD

**Issue by tgy3300** [gatsbyjs/gatsby#13237]
gatsby-image 插件不支持网络图片吗？比如图片地址为 https://www.aaa.com/g.jpg 的，怎么在gatsby-image插件中显示

**Comment by jonniebigodes** [comment]
@tgy3300 if you don't mind please use english as a language for the issues you're experiencing with Gatsby. As for instance google translator will help out non native speaking people, but there's always something lost in the translation and to avoid a miscomunication it would be better to go that route. Also to help out in your issue, fill the issue template with the information present if you don't mind. Feel free to make the adjustments i mentioned so that we can help you better.

**Comment by polarathene** [comment]
> Why do you want to ask this question, that is, I think that using the gatsby-image plugin to load images on a web page, the page will be much faster, and the browsing experience is also very good, the img tag is no experience like gatsby-image.

@tgy3300 yes that is what makes `gatsby-image` good. But it requires the image downloaded to process data. Use the plugin I linked, or download the image manually to query with GraphQL and use with `gatsby-image`. 

If you have not used `gatsby-image` before, try example project `gatsby new` in terminal.

@ tgy3300是的，这就是让'gatsby-image`变好的原因。 但它需要下载的图像处理数据。 使用我链接的插件，或手动下载图像以使用GraphQL查询并与`gatsby-image`一起使用。

如果您之前没有使用过`gatsby-image`，请在终端中尝试示例项目`gatsby new`。

**Comment by polarathene** [comment]
> BIG :+1: for this, BTW my native langugage is Chinese :grimacing: , I use the dictionary now and then to make sure my expression be understood and words are correct(I hope so :joy: )

@nihgwu Oh, I wasn't sure, your English is 真棒! 谢谢 for making the effort :smiley: 

> if you want to make use gatsby-image with remote images, you have to prepare a lot of props like scrSet which is not available for remote images

I have a project that uses that remote plugin I think, and it worked well. Images download and get processed like local ones, and queried via GraphQL. When I was in China(ChongQing), sometimes the images did not download in full, 130MB from Unsplash website. I used Netlify to download the images when it built the site which worked well.

**Comment by nihgwu** [comment]
@polarathene haha, I'm not saying the plugin doesn't work, I just want to clarify that the plugin actually downloads the images, so you are working with local images indeed, I said that you have to prepare the props is based on the assuption that you want to use the remote images directly
